xml/operations-integration_of_plugins_with_monasca-agent.xml

```diff
@@ -11,17 +11,17 @@
   Monasca-agent is an OpenStack open-source project. Monasca can also monitor
   non-openstack services. Third parties should install custom plugins into
   their &kw-hos-phrase; system using the steps outlined in the
-  <emphasis role="bold">Writing Custom Plugins</emphasis> section. If the
-  OpenStack community determines that the custom plugins are of general
-  benefit, the plugin may be added to the openstack/monasca-agent so that they
-  are installed with the monasca-agent. During the review process for
-  openstack/monasca-agent there are no guarantees that code will be approved or
-  merged by a deadline. Open-source contributors are expected to help with code
-  reviews in order to get their code accepted. Once changes are approved and
-  integrated into the openstack/monasca-agent and that version of the
-  monasca-agent is integrated with &kw-hos-phrase;, the third party can remove
-  the custom plugin installation steps since they would be installed in the
-  default monasca-agent venv.
+  <xref linkend="writing_custom_plugins"/> section. If the OpenStack community
```

Comment:
Remove "section" after the xref -- that would look weird since the link text is generated automatically.